src/pages/home/HeaderView.tsx

```diff
@@ -90,11 +90,12 @@ function HeaderView({
     const isSelfDM = ReportUtils.isSelfDM(report);
     const isGroupChat = ReportUtils.isGroupChat(report) || ReportUtils.isDeprecatedGroupDM(report);
     const isOneOnOneChat = ReportUtils.isOneOnOneChat(report);
+    const isSystemChat = ReportUtils.isSystemChat(report);
 
     // For 1:1 chat, we don't want to include currentUser as participants in order to not mark 1:1 chats as having multiple participants
     const participants = Object.keys(report?.participants ?? {})
         .map(Number)
-        .filter((accountID) => accountID !== session?.accountID || !isOneOnOneChat)
+        .filter((accountID) => accountID !== session?.accountID || (!isOneOnOneChat && !isSystemChat))
```

Comment:
Shouldn't this be `(!isOneOnOneChat || !isSystemChat)`?

Reply:
No, because `!isOneOnOneChat` will always be `true` for the systemChat, so we'd always end up including the current user in the participants list.